temoa/components/limits.py

```
@@ -175,7 +192,7 @@ def LimitDegrowthNewCapacityDeltaIndices(M: 'TemoaModel'):
 
 
 # @deprecated('Deprecated. Use LimitActivityGroupShare instead') # doesn't play well with pyomo
-def RenewablePortfolioStandard_Constraint(M: 'TemoaModel', r, p, g):
+def RenewablePortfolioStandard_Constraint(M: 'TemoaModel', r: Region, p: Period, g: str) -> Any:
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `RenewablePortfolioStandard_Constraint`

(ANN401)